app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,10 @@
+module ExceptionHandler
```

Comment:
@eclarizio @syncrou 
Should this code go in here where we have rest of the errors.

Reply:
@mkanoor Yeah, @syncrou and I talked about that, though that will be a separate PR since it is not directly related to fixing the linked issue.